twake/frontend/src/app/atoms/logo/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+type PropsType = {
+  size?: 16 | 32 | 64 | 128 | 256;
+}
+
+export const Logo = ({ size = 128 }: PropsType) => (
+  <div className="flex flex-col items-center justify-center">
```

Comment:
A good practice is to do this:
`interface PropsType extends Omit<React.HTMLAttributes<HTMLDivElement>, "size">{
  size?: 16 | 32 | 64 | 128 | 256;
}`

and then 

```
  <div className={"flex flex-col items-center justify-center " + props.className} {..._.omit(props, "className", "size")}>
```